env/README.md

```diff
@@ -7,41 +7,196 @@ compilation of the CROSSCON hypervisor.
 
 ## How to build the container
 
-### Build the docker image
+Run this command:
 
-Run the `./build.sh` to build the image. The resulting image will have
-the tag: `crosscon_hv`.
+```bash
+docker build -t crosscon_hv .
+```
 
-This will also produce the `build.log` log file, which is useful for
-debugging.
+to build the docker image. The resulting image will have a `crosscon_hv` tag.
+After the image has been built you will need to create a container from that
+image, it can be done with this command:
 
-### Create the container from the image
+```bash
+docker create --name crosscon_hv_container crosscon_hv
```

Comment:
Is this necessary when were using `docker run`? When following instructions i get:
```sh
$ docker create --name crosscon_hv_container crosscon_hv
0cd0057d3999e889df20d5b91cbc7d6f6f10b2c827d1920afc77089fc7eef1a6
$ docker run -d --name crosscon_hv_container crosscon_hv tail -f /dev/null
docker: Error response from daemon: Conflict. The container name "/crosscon_hv_container" is already in use by container "0cd0057d3999e889df20d5b91cbc7d6f6f10b2c827d1920afc77089fc7eef1a6". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
```

Reply:
That's true but the `docker create` is redundant and you can skip it and the error is caused by running `docker create` prior to `docker run`

Reply:
[`docker create`](https://docs.docker.com/reference/cli/docker/container/create/) creates a new container in a stopped state. To run this container one needs to use [`docker start`](https://docs.docker.com/reference/cli/docker/container/start/). [`docker run`](https://docs.docker.com/reference/cli/docker/container/run/) is basically a combination of these two commands.

Reply:
@tym2k1 you are right, i didn't understand at first. Adjusted the readme.md in [this commit](https://github.com/3mdeb/CROSSCON-Hypervisor-and-TEE-Isolation-Demos/pull/2/commits/ebc6b2a836cd6d1fa314bb0e73c6bc8231440166)